src/OpenApi/Serializer/OpenApiNormalizer.php

```diff
@@ -59,7 +59,7 @@ private function recursiveClean($data): array
             }
 
             // Side effect of using getPaths(): Paths which itself contains the array
-            if ('paths' === $key) {
+            if ('paths' === $key && \is_array($value) && isset($data['paths']['paths'])) {
```

Comment:
Shouldn't the `is_array` use `$data` and not `$value`?

Reply:
@alanpoulain granted. But if you look up at [line 57](https://github.com/api-platform/core/pull/4319/files#diff-66a6198b9d63f63ba3a635fa8f23143870ed8c73842fba6b142b1c1a118b4e96R57), the if statement also directly checks the `$value` contents to be an array. This leads us to the question: check for `is_array($value)` at all or just remove it?

Reply:
I think you can remove it.